packages/core/src/__tests__/ceramic-anchor.test.ts

```diff
@@ -314,30 +318,47 @@ describe('Ceramic anchoring', () => {
     await ceramic2.close()
   })
 
-  it('test the same state anchored twice, first one wins', async () => {
-    await ipfs2.swarm.connect(multaddr1)
+  it('test the same state anchored twice, last one wins', async () => {
```

Comment:
these changes completely change what this test is testing.  I'm actually not sure why removing nonces should have had any affect on this test at all?  Can we put it back the way it was?

Reply:
👍

Reply:
Which test duplicates this one?  This look like a basic conflict resolution test, the only one that exists at this high a level that I see.